packages/w3c-webdriver/test-env/d.ts

```diff
@@ -7,7 +7,6 @@ declare module 'iedriver' {
   export const path: string;
```

Comment:
Any specific reason we named this file as d?

Reply:
renamed to `declarations.ts`